### DISCUSSION TITLE
Crafting a Local RAG application with Quarkus

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#41197] [Quarkus Blog/Website]
# Crafting a Local RAG application with Quarkus

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/granite-rag/

**Comment by agarcia-github** [comment]
What a wonderful blog post! This is exactly what I need right now to start my journey into Quarkus and RAG. Thank you so much for posting.
